src/megachatapi.h

```diff
@@ -1941,31 +1938,47 @@ class MegaChatApi
     void disconnect(MegaChatRequestListener *listener = NULL);
 
     /**
-     * @brief Returns the current state of the connection
+     * @brief Returns the current state of the client
      *
      * It can be one of the following values:
      *  - MegaChatApi::DISCONNECTED = 0
      *  - MegaChatApi::CONNECTING   = 1
      *  - MegaChatApi::CONNECTED    = 2
      *
-     * @return The state of connection
+     * @note Even if this function returns CONNECTED, it does not mean the client
+     * is fully connected to chatd and presenced. It means the client has been requested
+     * to connect, in contrast to the offline mode.
+     * @see MegaChatApi::getChatConnectionState and MegaChatApi::areAllChatsLoggedIn.
+     *
+     * @return The connection's state of the client
      */
     int getConnectionState();
 
     /**
-     * @brief Returns the current state of the connection to chatd
+     * @brief Returns the current state of the connection to chatd for a given chatroom
      *
      * The possible values are:
      *  - MegaChatApi::CHAT_CONNECTION_OFFLINE      = 0
      *  - MegaChatApi::CHAT_CONNECTION_IN_PROGRESS  = 1
      *  - MegaChatApi::CHAT_CONNECTION_LOGGING      = 2
      *  - MegaChatApi::CHAT_CONNECTION_ONLINE       = 3
      *
+     * The connection state of all chats can be checked by passing MEGACHAT_INVALID_HANDLE
```

Comment:
I think getChatConnectionState returns MegaChatApi::CHAT_CONNECTION_OFFLINE if MEGACHAT_INVALID_HANDLE is passed as chatid

Reply:
You're correct. It was a leftover from the initial approach.
Fixed.